profile/forgot-pass.html

```diff
@@ -0,0 +1,86 @@
+    <link rel="stylesheet" href="https://bloom-component-library.vercel.app/components/component.css">
+    <script src="https://kit.fontawesome.com/00d6549dc7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="profile.css">
+    <link rel="stylesheet" href="../home.css">
+    <title>Bloom Tronics | Forgot Password</title>
+</head>
+<body>
+    
+    <nav class="nav">
+        <div class="nav-main">
+            <div class="nav-logo">
+                <a href="../index.html" class="logo-link">
+                    <img class="logo-img" src="../images/logo.png" alt="logo">
+                    <h1 class="nav-header">Bloom Tronics</h1>
+                </a>
+            </div>
+            <div class="nav-search">
+                <input type="text" class="search-box" placeholder="Search">
+                <button class="search-btn"><i class="fa-solid fa-magnifying-glass"></i></button>
+            </div>
+            <div class="nav-items">
+                <a href="./login.html" class="sign-in">
+                    <i class="fa-solid fa-circle-user"></i>
+                </a>
+                <div class="icon cart-badge">
+                    <a href="../Wishlist/wishlist.html">
+                        <i class="fa-solid fa-heart"></i>
+                        <div class="notification-icon flex-center">
+                            <span>3</span>
+                        </div>   
+                    </a>
+                </div> 
+                <div class="icon cart-badge">
+                    <a href="../Cart/cart.html">
+                        <i class="fa-solid fa-cart-shopping"></i>
+                        <div class="notification-icon flex-center">
+                            <span>2</span>
+                        </div>
+                    </a>
+                </div>
+            </div>
+            </div>
+        </div>
+        <ul class="links-container">
+            <li class="link-item"><a href="#" class="link">TV's Audio , Video</a></li>
+            <li class="link-item"><a href="#" class="link">Smartphone & Gadget</a></li>
+            <li class="link-item"><a href="#" class="link">Computer & Laptops</a></li>
+            <li class="link-item"><a href="#" class="link">Home Appliances</a></li>
+            <li class="link-item"><a href="#" class="link">accessories</a></li>
+        </ul>
+    </nav>
+
+    <div class="profile-container">
+        <div class="profile-main-container">
+            <div class="profile-header ">
+                <img class="logo-img" src="../images/logo.png" alt="logo">
+                <h2>Forgot Password</h3>
+            </div>
+            <div class="profile-main">
+
+                    <div class="profile-email">
+                        <label for="mail">Enter your email and we will send you a link to reset your password.</label>
+                        <input type="text"placeholder='Email' class="input-text">
+                    </div>
```

Comment:
You can add id attribute to the input element

Reply:
Yes will change it !! Thanks